applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -567,15 +567,13 @@ void ConstitutiveLawUtilities<6>::CalculateProjectionOperator(
     indexes_ij(0, 1) = 1;
     indexes_ij(1, 0) = 1;
     indexes_ij(1, 1) = 2;
-    indexes_ij(2, 0) = 0;
-    indexes_ij(2, 1) = 2;
```

Comment:
Sorry I think this was right before.

Reply:
yes! I think that this: 

for (IndexType i=0; i< Dimension - 1; ++i) {
for (IndexType j=i + 1; j< Dimension; ++j) {

is the same that the code in the begining.